Invalid test command sample message

Following README instructions https://github.com/LayerZero-Labs/gasolina-gcp/blob/f089c465a8368375705fe002236571826acff835/README.md?plain=1#L109-L113

It returns an error
```
Error: Cannot find module './testDeployment'
```

--- a/README.md
+++ b/README.md
@@ -109,7 +109,7 @@ If everything is successful, you should have a running gasolina-api deployed to 
 3. To test the API against a sample message, in the root directory run:
 
 ```bash
-ts-node scripts/testDeployment -u <URL> -e <environment>
+ts-node scripts/testDeployment.ts -u <URL> -e <environment>
 ```
 
 -   A successful response will look like:

